src/Chapter10/Listing10.23.RegisteringAFinalizerWithProcessExit.cs

```diff
@@ -93,12 +94,14 @@ public SampleUnmanagedResource()
             WriteLine("Exiting...");
         }
 
-        #region IDisposable Members
         public void Dispose()
         {
             Dispose(true);
+            
+            // Unregister from the finalization queue.
```

Comment:
I realize this matches the original comment, but I think I would change this wording since the idea of the finalization queue is an implementation detail that is only mostly true (since different runtimes may change this). Also, at this point in the code the object has not been add to it, instead it is preventing itself from getting added.

Perhaps something more like this (stollen from the MS docs):
```suggestion
            // Request that the finalizer not be called for this object.
```

Reply:
Sounds great to me!